domain/flag/state/state.go

```diff
@@ -30,66 +30,73 @@ func NewState(factory coredb.TxnRunnerFactory, logger logger.Logger) *State {
 
 // SetFlag sets the value of a flag.
 // Description is used to describe the flag and it's potential state.
-func (s *State) SetFlag(ctx context.Context, flag string, value bool, description string) error {
+func (s *State) SetFlag(ctx context.Context, flagName string, value bool, description string) error {
 	db, err := s.DB()
 	if err != nil {
 		return errors.Trace(err)
 	}
 
-	query := `
-INSERT INTO flag (name, value, description)
-VALUES (?, ?, ?)
+	f := dbFlag{Name: flagName, Value: value, Description: description}
```

Comment:
**suggestion:** `f` may be too short as name for a variable with a scope as large as a function. Moreover, in idiomatic golang, [we tend to use one-char name for receiver](https://go.dev/wiki/CodeReviewComments#receiver-names).

A good reading about naming variables: https://google.github.io/styleguide/go/decisions#variable-names

side note: This comment can be done for other usage of `f` for flag, in a lesser extent, for `stmt` as `statement`.

Reply:
Ah yea, had meant to change this to `flag` after chaining the parameter name from `flag` to `flagName`, thanks

Reply:
You miss an occurence l. 80 ;)

Reply:
Got it, thanks!